config/apertures.py

```diff
@@ -0,0 +1,5 @@
+# Set up aperture photometry
+# 'config' should be a SourceMeasurementConfig
+
+# Use a large aperture to be independent of seeing in calibration
+config.plugins["base_CircularApertureFlux"].maxSincRadius = 12.0
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)